src/styles.css

```diff
@@ -0,0 +1,22 @@
+.form-container {
+  width: 30rem;
+  padding: 2rem;
+  position: absolute;
+  top: 10rem;
+  left: 20rem;
+  border: 0.1rem solid;
```

Comment:
Use Bootstrap's colors. Avoid choosing colors (especially pure black). Try to make your app look like the Bootstrap Docs. The CSS framework already made a bunch of choices that go together, so you don't want to deviate from them. If you want something a little more custom, you can choose a theme from here: https://bootswatch.com/
And add the CSS from here: https://cdnjs.com/libraries/bootswatch